packages/editor-ui/src/components/FilterConditions/Condition.vue

```diff
@@ -261,12 +273,20 @@ const onBlur = (): void => {
 
 .statusIcon {
 	padding-left: var(--spacing-4xs);
+	padding-right: var(--spacing-4xs);
 }
 
 .remove {
 	position: absolute;
 	left: 0;
-	top: var(--spacing-l);
+	top: calc(14px + var(--spacing-m));
+	opacity: 0;
+	transition: opacity 100ms ease-in;
+}
+.drag {
+	position: absolute;
+	left: 0;
+	top: var(--spacing-m);
 	opacity: 0;
 	transition: opacity 100ms ease-in;
```

Comment:
I think `.drag` and `.remove` could be reduced to 1 class to avoid duplication

Reply:
updated